app/controllers/admin/members_controller.rb

```diff
@@ -35,4 +39,47 @@ def permitted_params
     params.require(:member).permit %i(email username approved first_name
     last_name phone postal_street postal_zip postal_city)
   end
+
+  def load_website
+    if current_user.admin?
+      @website = current_website
+    else
+      @website = current_user.websites.find(current_website.id)
```

Comment:
Can a user try to load another site and get access to it with this command?

here, It tries to set current_website with subdomin, or domain --> https://github.com/standout/standoutcms/blob/master/app/controllers/application_controller.rb#L95

We can i guess do something like current_user.website i guess?